README.md

```diff
@@ -68,14 +49,6 @@ To use web version of this app with your local server:
    to enable CORS
 2. Run local devserver
 
-### Future?
-
-I'd love Systers to accept this project as one of theirs. I do realize it requires
-better documentation and has to be polished a bit, but I really believe these problems
-are easy to solve.
-My dream? I'll be incredibly happy, for example, if students in next year's Google Code-in will continue its development,
-improve it and polish the codebase.
-
 ### Contact us
```

Comment:
Above this section you could add a new section:

> Maybe you can add a Contributing section here and include a text to highlight the CONTRIBUTING guidelines that should be read before starting to contribute. Also the reporting guidelines and code of conduct

You can check how it is on Mentorship Android for example